### ISSUE TITLE
A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader : No proper name is announced for "Close" control for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11528]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#A11ySev2;#Benchmark;#MAS1.3.1;#MAS4.1.2;#FTP;

### Environment Details:
Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf5fe8f60d04d70096bc94c654cab60d81a
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS4.1.2

### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3. Navigate to the "Close" (X) control present besides "Start Tour" control.
4. Start screen reader, listen if proper name is announced for the "Close" control or not.

### Actual:
When screen reader user navigate to the "Close" (X) control only the role of the control i.e. "button" is announced for the screen reader user, but no proper name as "Close" is announced so that the user is able to identify the control.

### Expected:
The "Close" (X) control should be announced with a meaningful name as "close" for screen reader users.

### User Impact:
If the proper name of the interactive element is not announced for screen reader users then the user will have difficulty in understanding the control and interacting with the control.

### Attachment link for Reference:
[11528_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader No proper name is announced for "Close" control for screen reader users.mov.zip](https://github.com/microsoft/azuredatastudio/files/4980911/11528_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader.No.proper.name.is.announced.for.Close.control.for.screen.reader.users.mov.zip)

**Comment by Ashutoshsahoo164** [comment]
Closing this bug , as the scenario itself is not present in UI.
GitHubTags:#Closed;
